src/bridge.cpp

```diff
@@ -175,31 +174,33 @@ coord_t evaluateBridgeLine(const coord_t line_y, const TransformedShape& transfo
 
         case BridgeStatus::Supported:
             bridge_status = leaving_skin ? BridgeStatus::Outside : BridgeStatus::Anchored;
+            // Negatively account for fully supported lines to avoid lonely line parts over the supported areas
+            add_segment_score_weight = -0.1;
```

Comment:
The `-0.1` value seems a bit arbitrary (I even thought it might be a typo for `-1.0` at first; until I took a closer look at what it replaced and why you might want to use a double at all...)

Reply:
It is very much arbitrary indeed... The previous version was mostly working, with an equivalent score of 0, compared to scores 1.0 and -1.0 for other lines. But on one edge case I had to add a negative value for this type of line. I tried with -1.0, which worked but felt a bit wrong. -0.1 also worked and feels less harmful. If there are other annoying edge cases, we can stil adjust it further, but so far all the tested cases work as expected.